### PR TITLE
Ng 19 1024x650 window

### PR DESCRIPTION
**Pull request by alejoreyes96**
Added GUI folder and window file. Running main.py creates a window with sample editable data tables, useful for later use.

### CODE REVIEW

File: GUI/window.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+from kivy.uix.behaviors import FocusBehavior
+from kivy.uix.recycleview.layout import LayoutSelectionBehavior
+from kivy.uix.popup import Popup
+from kivy.config import Config
+Config.set('graphics', 'width', '1024')
+Config.set('graphics', 'height', '650')
+Config.set('graphics', 'resizable', False)
+
+# Window tables with editable data rows
+#Uses test.kv as a config file
```
Comment (ChristianPerez34) [inline GUI/window.py:17]: test.kv is only for testing purposes or is it the actual config file? 

If it is the config file then it should be renamed as such

File: GUI/window.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+class TextInputPopup(Popup):
+    obj = ObjectProperty(None)
+    obj_text = StringProperty("")
+
+    def __init__(self, obj, **kwargs):
+        super(TextInputPopup, self).__init__(**kwargs)
```
Comment (ChristianPerez34) [inline GUI/window.py:24]: Why do you not pass obj to super?

**Review by cfboy — APPROVED** [review]

**Review by noelvalentin — APPROVED** [review]

File: GUI/window.py (hunk 1 above)
Comment (alejoreyes96) [reply GUI/window.py:17]: It is not a permanent config file, just for the display of the current tables/columns. However, it won't be necessarily needed later on. It is an example/placeholder of how to configure the window.

File: GUI/window.py (hunk 2 above)
Comment (alejoreyes96) [reply GUI/window.py:24]: Should I? I thought that by passing the class `TextInputPopup` was enough.

Comment (ChristianPerez34) [reply GUI/window.py:24]: Does the super class also have `obj` or is that something you defined?

Comment (alejoreyes96) [reply GUI/window.py:24]: It's something I defined, and I'm just initializing it under the super passing. This obj is used to read the content when editing the rows of data on the window. Technically it is not needed, but it is a nice feature to know about to modify it later on.

**Review by ChristianPerez34 — APPROVED** [review]

**Pull request merged**